sync_ratings.py

```diff
@@ -91,16 +91,21 @@ def setup_logging(self):
 			self.logger.addHandler(ch_std)
 
 	def sync(self):
+		if self.options.sync not in ["both", "tracks","playlists"]:
```

Comment:
Your help says that upper-case is expected. I think allowing any case, even `PlaYLisTs` should be possible
```suggestion
		if self.options.sync.lower() not in ["both", "tracks","playlists"]:
```